src/registrar/admin.py

```diff
@@ -162,12 +185,16 @@ def response_change(self, request, obj):
 
 
 class ContactAdmin(ListHeaderAdmin):
```

Comment:
Doesn't ContactAdmin inherit from ListHeaderAdmin which inherits from AuditedAdmin? Therefore no need to repeat def formfield_for_foreignkey?

Reply:
Good catch